### DISCUSSION TITLE
Convert styled component to sx props

### DISCUSSION THREAD

**Discussion by justoverclockl** [system-ui/theme-ui#2375] [General] [answered]

**Comment by lachlanjc** [comment, marked as answer]
Do you mean you're using MUI? Theme UI invented the `sx` attribute (afaik) & we're thrilled MUI has adopted it, but MUI's repo is [over here](https://github.com/mui/material-ui). Also not sure what your question exactly is?

**Comment by justoverclockl** [comment]
Ops sorry!
